ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/molecule.yml

```diff
@@ -0,0 +1,48 @@
+---
+scenario:
+  converge_sequence:
+    - syntax
+    - create
+    - converge
+    - verify
+  test_sequence:
+    - syntax
+    - create
+    - converge
+    - idempotence
+    - verify
+  cleanup_sequence:
+    - destroy
+dependency:
+  name: galaxy
+driver:
+  name: delegated
+platforms:
+  - name: DC1-LEAF1A
+    image: avdteam/base:3.8-v2.0
+    pre_build_image: true
+    managed: false
+    groups:
+      - DC1_LEAF1
+      - DC1_LEAFS
+      - DC1_FABRIC
+      - AVD_LAB
```

Comment:
we probably should change this

Reply:
I wish we could just remove this, I have update with relevant information.
Other seggestions are welcome!

Reply:
It can be removed. Check how it's done in `eos_cli_config_gen_deprecated_vars`. You just add a dummy here and then target a specific group in the playbooks instead of `all`